aii-ks/src/main/perl/ks.pm

```diff
@@ -692,8 +692,14 @@ sub log_action {
     }
 
     if ($consolelogging) {
+        # In EL7, /dev/console doesn't add the carriage return for each line feed (new line)
+        # resulting in unreadable messages on the console. /dev/pts/0 must be used instead.
+        # But on prior versions, /dev/pts/0 doesn't exist at installation time and cannot be used.
+        # The following code allows to use /dev/pts/0 if it exists else to revert to /dev/console.
+        push(@logactions, "console='/dev/console'");
```

Comment:
there is no harm in always setting these variables early on, even if no consolelogging is used. this will allow a cleaner if/else construct, because this is a bit ugly.